### PR TITLE
Add missing test fields for TestAccVcdNsxtEdgeBgpConfigIntegrationVdc and TestAccVcdNsxtEdgeBgpConfigIntegrationVdcGroup

### PR DESCRIPTION
**Pull request by Didainius**
Tests `TestAccVcdNsxtEdgeBgpConfigIntegrationVdc` and `TestAccVcdNsxtEdgeBgpConfigIntegrationVdcGroup` had missing `Name` field. As a result binary tests would end up with `name` being `name = "*** MISSING FIELD [TestName] from func vcd.TestAccVcdNsxtEdgeBgpConfigIntegrationVdcGroup-1"`.

This PR adds missing variables.

Tests still pass.

### CODE REVIEW

**Review by adambarreiro — COMMENTED** [review]
Thanks for the fix! Should we add a changelog entry for this one?

**Review by dataclouder — APPROVED** [review]

**Comment by Didainius** [comment]
> Should we add a changelog entry for this one?

Yes. Added - thanks

**Review by adambarreiro — APPROVED** [review]

File: .changes/v3.9.0/958-notes.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+* Add missing test fields for TestAccVcdNsxtEdgeBgpConfigIntegrationVdc and
```
Comment (lvirbalas) [inline .changes/v3.9.0/958-notes.md:1]:
```suggestion
* Add missing test name fields for TestAccVcdNsxtEdgeBgpConfigIntegrationVdc and
```

**Review by lvirbalas — APPROVED** [review]
One suggestion, thanks!

File: .changes/v3.9.0/958-notes.md (hunk 1 above)
Comment (Didainius) [reply .changes/v3.9.0/958-notes.md]: fixed

**Pull request merged**